onecodex/viz/_distance.py

```diff
@@ -29,43 +27,94 @@ def plot_distance(analyses, metric='braycurtis',
         raise OneCodexException("'metric' must be one of "
                                 "braycurtis, manhattan, jaccard, or unifrac")
 
-    normed_classifications, metadata = normalize_classifications(analyses, label=label)
-    if len(normed_classifications) < 2:
-        raise OneCodexException('`plot_distance` requires 2 or more valid classification results.')
+    if rank is None:
```

Comment:
`rank` is set by default above so this seems a little redundant?

Reply:
True but if rank winds up being None it with normalize=True, it will wind up normalizing readcount_w_children across all ranks and be meaningless. It's only to make it more difficult for users to get bad results.